virtualization-and-containers/material/singularity_slides.md

```diff
@@ -0,0 +1,181 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+  font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Singularity
+
+<img src="https://sylabs.io/assets/svg/singularity-logo.svg" width=20%; style="margin-left:auto; margin-right:auto; padding-top: 25px; padding-bottom: 25px">
+
+[https://sylabs.io/assets/svg/singularity-logo.svg](https://sylabs.io/assets/svg/singularity-logo.svg)
+
+---
+
+## Introduction
+
+- Initiated at Lawrence Berkeley National Laboratory
+- Now: Developed by [Sylabs](https://sylabs.io)
+- Written in  Go
+- Comes in [different flavors](https://sylabs.io/singularity/)
+  - SingularityCE (Community edition, open source, BSD licensed)
+  - SingularityPro (Enterprise)
+  - Singularity Enterprise (More enterprise?)
+  - SingularityDesktop (Mac, Beta)
+  - [Sylabs Cloud](https://cloud.sylabs.io/library)
+    - Image registry and remote builder
+  - [Singularity Hub](https://singularityhub.github.io)
+    - Open source registry
+- Community edition open source (BSD license)
+
+---
+
+## Goals
+
+- Verifiable reproducibility and security
+  - Immutable images with encryption and verification support
+- Integration over isolation by default
+  - Access hardware (GPU, network...) and filesystems
+- Simple and effective security model
+  - Same user inside and outside container
+- Mobility of Compute
```

Comment:
of Compute? no understand

Reply:
That is a term the Singularity people like. I will put in quotation marks.